.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
ion":true,"phpdoc_single_line_var_spacing":true,"phpdoc_tag_type":{"tags":{"inheritDoc":"inline"}},"phpdoc_to_comment":true,"phpdoc_trim":true,"phpdoc_trim_consecutive_blank_line_separation":true,"phpdoc_types":true,"phpdoc_types_order":{"null_adjustment":"always_last","sort_algorithm":"none"},"phpdoc_var_without_name":true,"semicolon_after_instruction":true,"single_class_element_per_statement":true,"single_line_comment_style":{"comment_types":["hash"]},"single_line_throw":true,"single_quote":true,"single_space_after_construct":true,"space_after_semicolon":{"remove_in_empty_for_expressions":true},"standardize_increment":true,"standardize_not_equals":true,"switch_continue_to_break":true,"trailing_comma_in_multiline":true,"trim_array_spaces":true,"types_spaces":true,"unary_operator_spaces":true,"whitespace_after_comma_in_array":true,"blank_line_after_opening_tag":true,"compact_nullable_typehint":true,"declare_equal_normalize":true,"lowercase_cast":true,"lowercase_static_reference":true,"new_with_braces":true,"no_blank_lines_after_class_opening":true,"no_leading_import_slash":true,"no_whitespace_in_blank_line":true,"ordered_class_elements":{"order":["use_trait"]},"return_type_declaration":true,"short_scalar_cast":true,"single_blank_line_before_namespace":true,"single_trait_insert_per_statement":true,"ternary_operator_spaces":true,"visibility_required":true,"blank_line_after_namespace":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"no_break_comment":true,"no_closing_tag":true,"no_space_around_double_colon":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"encoding":true,"full_opening_tag":true,"error_suppression":{"mute_deprecation_error":false,"noise_remaining_usages":false,"noise_remaining_usages_exclude":[]},"non_printable_character":true,"header_comment":{"comment_type":"PHPDoc","header":"Copyright (c) Since 2020 Friends of Presta\n\nNOTICE OF LICENSE\n\nThis source file is subject to the Academic Free License (AFL 3.0)\nthat is bundled with this package in the file docs\/licenses\/LICENSE.txt.\nIt is also available through the world-wide-web at this URL:\nhttps:\/\/opensource.org\/licenses\/afl-3.0.php\nIf you did not receive a copy of the license and are unable to\nobtain it through the world-wide-web, please send an email\nto infos@friendsofpresta.org so we can send you a copy immediately.\n\n@author    Friends of Presta <infos@friendsofpresta.org>\n@copyright since 2020 Friends of Presta\n@license   https:\/\/opensource.org\/licenses\/AFL-3.0  Academic Free License (\"AFL\") v. 3.0\n","location":"after_open","separate":"bottom"}},"hashes":{"src\/Commands\/Module\/ModuleGenerate.php":2870719447}}
```

Comment:
This file shoud not be commited.
We could add it in the .gitignore file.

Reply:
fixed